src/torchcodec/encoders/_video_encoder.py

```diff
@@ -51,18 +51,23 @@ def to_file(
             codec (str, optional): The codec to use for encoding (e.g., "libx264",
                 "h264"). If not specified, the default codec
                 for the container format will be used.
+                See :ref:`codec_selection` for details.
             pixel_format (str, optional): The pixel format for encoding (e.g.,
                 "yuv420p", "yuv444p"). If not specified, uses codec's default format.
+                See :ref:`pixel_format` for details.
             crf (int or float, optional): Constant Rate Factor for encoding quality. Lower values
                 mean better quality. Valid range depends on the encoder (commonly 0-51).
```

Comment:
Here and for the other docstrings, I think we should rather say


```suggestion
                mean better quality. Valid range depends on the encoder (e.g.  0-51 for libx264).
```